addons/binding/org.openhab.binding.robonect/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,289 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions
+		bindingId="robonect" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+		xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="mower">
+		<label>Mower</label>
+		<description>Mower robot connected via robonect module</description>
+		<channel-groups>
+			<channel-group id="mowerInfo" typeId="mowerInfoType"/>
+			<channel-group id="mowerStatus" typeId="mowerStatusType"/>
+			<channel-group id="timer" typeId="timerType"/>
+			<channel-group id="wlan" typeId="wlanType"/>
+			<channel-group id="job" typeId="jobGroupType"/>
+			<channel-group id="error" typeId="errorGroupType"/>
+			<channel-group id="lastError" typeId="errorGroupType"/>
+			<channel-group id="health" typeId="healthType"/>
+		</channel-groups>
+		<properties>
+			<property name="vendor">robonect.de</property>
+		</properties>
+		<config-description>
+			<parameter name="host" type="text" required="true">
+				<context>network-address</context>
+				<label>Host</label>
+				<description>Host name or network address of the Robonect module</description>
+			</parameter>
+			<parameter name="user" type="text" required="false">
+				<label>User</label>
+				<description>The user id if authentication has been enabled on the mower</description>
+			</parameter>
+			<parameter name="password" type="text" required="false">
+				<context>password</context>
+				<label>Password</label>
+				<description>The password if authentication has been enabled on the mower</description>
+			</parameter>
+			<parameter name="pollInterval" type="integer" required="false" unit="s">
+				<label>Polling interval</label>
+				<default>30</default>
+				<description>The interval for the binding to poll the mowers status information.
+					seconds.
+				</description>
+			</parameter>
+			<parameter name="offlineTimeout" type="integer" required="false" unit="m">
+				<label>Offline timeout</label>
+				<default>30</default>
+				<description>The maximum time the mower may be offline before the offline trigger is triggered.
+				</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-group-type id="mowerInfoType">
+		<label>Mower Info</label>
+		<channels>
+			<channel id="name" typeId="nameType"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="mowerStatusType">
+		<label>Mower Status</label>
+		<channels>
+			<channel id="battery" typeId="system.battery-level"/>
+			<channel id="duration" typeId="durationType"/>
+			<channel id="hours" typeId="hoursType"/>
+			<channel id="mode" typeId="modeType"/>
+			<channel id="status" typeId="mowerStatusType"/>
+			<channel id="started" typeId="startedType"/>
+			<channel id="offlineTrigger" typeId="offlineTriggerType"/>
```

Comment:
Well, the idea here is, that the binding provides functionality of allowing certain duration of "offline" and trigger an alert if this is tine exceeded. The real world use case is, that in many cases the mower actually looses the signal to Wifi as it exits the range but finally will come back. If not, most probably the mower sticks somewhere. Now the "community" at robonect.de wished the functionality in the binding to not have to code custom rules for this condition. I'm happy to remove it, if you provide a sample rule which handles this use case properly. Then I can put this in the Readme.md...

Reply:
Okay, I see. 

With the new rule engine it would be simple as the binding could bring a `TriggerModule` which encapsulates exactly this functionality. But as most of us are still stuck on the old one, I agree it would be more convenient to keep this channel.